web/src/pages/onboarding.tsx

```diff
@@ -161,7 +161,7 @@ const OnboardingPage = (props: Props): ReactElement => {
         return;
       }
 
-      let currentUserData = updateQueue?.current();
+      let currentUserData = userData;
```

Comment:
It looks like `currentUserData` was created to be more friendly than `updateQueue?.current()`. Can we just use `userData` in all of these locations?

Reply:
We actually can't, because this is onboarding we might need to create user data for new users (see line 170)